bids/version.py

```diff
@@ -45,7 +45,7 @@
 AUTHOR_EMAIL = "bids-discussion@googlegroups.com"
 PLATFORMS = "OS Independent"
 # No data for now
-REQUIRES = ["grabbit>=0.2.3", "six", "num2words", "numpy", "scipy", "pandas",
+REQUIRES = ["grabbit==0.2.4", "six", "num2words", "numpy", "scipy", "pandas",
```

Comment:
`>=`?

Reply:
I don't see a reason not to pin versions. It's probably a good idea to increment pybids version anyway in cases where grabbit functionality changes non-trivially, and this way we avoid any nasty surprises.